### PR TITLE
Add release branches to the playbooks for release 24.7

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
```[tasklist]
### Fixes
- [ ] https://github.com/stackabletech/demos/pull/75
- [ ] https://github.com/stackabletech/hive-operator/pull/496
- [ ] https://github.com/stackabletech/spark-k8s-operator/pull/441
- [ ] todo: add above getting-started fixes to main branch (to maintain the intended consistency) @NickLarsenNZ
- [ ] https://github.com/stackabletech/documentation/pull/636
```

### CODE REVIEW

File: scripts/publish-new-version.sh [hunk 1]
```diff
@@ -145,7 +145,7 @@ publish_branch="publish-$docs_version"
 git checkout -b "$publish_branch"
 
 git add .
-git commit -m "Add release branches to the playbooks for release $docs_version."
+git commit -m "chore: Add release branches to the playbooks for release $docs_version"
```
Comment (nightkr) [inline scripts/publish-new-version.sh:148]: :/

**Review by nightkr — DISMISSED** [review]

File: scripts/publish-new-version.sh (hunk 1 above)
Comment (NickLarsenNZ) [reply scripts/publish-new-version.sh:148]: Conventional Commits ;)
Embrace them

**Comment by NickLarsenNZ** [comment]
hmm, there's an error that perhaps @fhennig needs to check out
https://github.com/stackabletech/documentation/actions/runs/10165503435/job/28113439427#step:5:22

File: scripts/publish-new-version.sh (hunk 1 above)
Comment (NickLarsenNZ) [reply scripts/publish-new-version.sh:148]: Oh, but I see. This is the docs repo and I think we didn't care about CCs in this particular repo.

It's not too late to revert that, as there are other errors that seem to need fixing.

Comment (NickLarsenNZ) [reply scripts/publish-new-version.sh]: Reverted in 67e973b

**Review by lfrancke — APPROVED** [review]

**Pull request merged**